Gemfile

```diff
@@ -17,7 +17,7 @@ gem 'jquery-rails'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
-gem 'acts-as-taggable-on', '~> 4.0'
+gem 'acts-as-taggable-on', '~> 5.0'
```

Comment:
As your first commit is to add this gem, if this is just to use a newer version, you should fixup this change to the first commit.